aurora_echo/echo_clone.py

```diff
@@ -76,7 +76,7 @@ def collect_clone_params(source_cluster_name: str, new_cluster_name: str, db_sub
 
 
 def collect_instance_params(cluster_identifier: str, new_instance_name: str, engine: str, db_instance_class: str,
-                            availability_zone: str, tags: list):
+            availability_zone: str, tags: list, db_parameter_group_name: str):
```

Comment:
looks like github dropped my comment but I tried to say

> The removal of this whitespace is not PEP8. The 2nd line of function parameters should align with the first parameter 

with that review

Reply:
@whwright Thank you. I will fix it.